UniAtHome/UniAtHome.DAL/UniAtHomeDbContext.cs

```diff
@@ -1,20 +1,20 @@
 ﻿using System;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore;
 using UniAtHome.DAL.Entities;
 
 namespace UniAtHome.DAL
 {
-    public class UniAtHomeDbContext : DbContext
+    public class UniAtHomeDbContext : IdentityDbContext<User>
     {
         public UniAtHomeDbContext(DbContextOptions<UniAtHomeDbContext> options) : base(options)
         {
         }
 
         public DbSet<Course> Courses { get; set; }
         public DbSet<Lesson> Lessons { get; set; }
-        public DbSet<User> Users { get; set; }
```

Comment:
Why did you remove this?

Reply:
Context now extends identity context, so it has DbSet<User> inherited from the parent